pkg/server/grpc_server.go

```diff
@@ -140,6 +140,7 @@ func (g *grpcCAServer) CreateSigningCertificate(ctx context.Context, request *fu
 		// currently configured CA doesn't support pre-certificate flow required to embed SCT in final certificate
 		csc, err = g.ca.CreateCertificate(ctx, principal, publicKey)
 		if err != nil {
+			logger.Error("Error creating certificate with CA backend: ", err)
```

Comment:
Perhaps just wrap `err` with this string - a log statement will fire in the subsequent call to `handleFulcioGRPCError`

same comment in other cases below.

Reply:
True! I did that for those whose error messages are similar to the fired error. There are few that I kept so they return  descriptive error message to easily distinguish where they are occurring.

Reply:
Sorry, after re-reading my comment I think I could have worded it better.

My thought was that within each of those `if err != nil` conditions, a log message will be generated to reflect the error (within the call to `handleFulcioGRPCError`). I agree that the errors could be scoped to be more useful, but with your change we'll have two log entries with slightly different error messages which might confuse someone. 

I was thinking of something like:

```
if err != nil {
    err = fmt.Errorf("Error creating certificate with CA backend: %w", err)
    // all code as is below
    // ...
}
```

That way you get the specificity you're looking for without the duplicate log message. Curious to hear your thoughts on that approach.

Reply:
I can change that to avoid duplicating log errors. However I'd like to keep some custom err messages to be consistent with the current implementation of other error types, e.g. invalidIdentityToken, invalidCSR error messages.